src/hooks.ts

```diff
@@ -131,7 +131,7 @@ export function useProgress(updateInterval = 1000) {
     return () => {
       mounted = false;
     };
-  }, [playerState, updateInterval]);
+  }, [isNone, updateInterval]);
 
   return state;
```

Comment:
@puckey would doing a:

```
return state || State.None;
```

Allow us to avoid the breaking change here?

Reply:
Not in this way, but there is slightly more convoluted way to fix this.. Will take a look at it later